bobtimus/src/lib.rs

```diff
@@ -270,32 +275,92 @@ where
     /// We return the range of possible loan terms to the borrower.
     /// The borrower can then request a loan using parameters that are within our terms.
     pub async fn handle_loan_offer_request(&mut self) -> Result<LoanOffer> {
```

Comment:
Does not look like it needs to be async anymore.

Reply:
There is a lint you can enable to catch these automatically: https://rust-lang.github.io/rust-clippy/master/index.html#unused_async